.buildkite/pipeline.yml

```
@@ -130,6 +131,7 @@ steps:
 
     - label: Run Haskell E2E Tests (linux)
       command: 'nix develop --command bash -c "just e2e-local"'
+      timeout: 5
```

```suggestion
      timeout_in_minutes: 5
```

The Buildkite help page doesn't mention `timeout` anymore, only `timeout_in_minutes`. 🤔

---

Changing everywhere